designsafe/static/scripts/ng-designsafe/providers/notifications-provider.js

```diff
@@ -64,9 +64,11 @@ function NotificationService(
      * @param {Object} msg
      */
     function processMessage(e, msg) {
-        processors.notifs.process(msg);
         $rootScope.$broadcast('notification', msg);
         const eventType = msg.event_type.toLowerCase();
+        if (eventType === 'job' || eventType === 'interactive_session_ready' || eventType === 'WEB') {
```

Comment:
I don't see `WEB` appear anyone else in the changes. Is that expected?

Reply:
Removed, used to be an event type but no longer